eras/byron/ledger/impl/cardano-ledger-byron.cabal

```diff
@@ -347,7 +347,6 @@ test-suite cardano-ledger-byron-test
         directory,
         filepath,
         formatting,
-        generic-monoid,
```

Comment:
Changes like this requires a patch version bump of the package, otherwise it will not get released. So, could you bump it to `1.0.0.4`